python_benchmarks/20_polars_lazy.py

```diff
@@ -1,18 +1,15 @@
-import polars
+import polars as pl
 
 
 def polars_lazy(fname):
-    dtypes = {f"column{i}": (polars.UInt32 if i == 0 else polars.Float32())
+    dtypes = {f"column{i}": (pl.UInt32 if i == 0 else pl.Float32())
               for i in range(9)}
-    return (polars.scan_csv(fname, dtypes=dtypes)
-                  .select(
-                      polars.sum_horizontal(polars.all()
-                                                  .exclude("0")
+    return (pl.scan_csv(fname, dtypes=dtypes)
+                  .filter(pl.sum_horizontal(pl.all()
+                                                  .exclude("column0")
                                                   .pow(2)
-                      ).sqrt()
-                       .alias("sum_sq"))
-                  .filter(polars.col("sum_sq") < 1e16)
-                  .count()
+                      ).sqrt() < 1e16)
+                  .select(pl.len())
```

Comment:
This is Polars' `SELECT count(*) ...` query, and that way we are not forced to materialize the other columns. (currently we still do, but are not forced).